west.yml

```diff
@@ -96,7 +96,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 25b9cf5b65c9bafbe8e885b6ad155bb56e2df460
+      revision: 3a78a5e2dea65234f4bc3a16a59bacade637484a
```

Comment:
@b-gent You should use `pull/179/head` ref for now - west is not able to find SHA from your branch.

Reply:
Thanks, I was assuming this has to be updated anyway after the nrfxlib is merged. Changed now.